test/sourcemaps/index.ts

```diff
@@ -31,7 +31,8 @@ describe('sourcemaps', () => {
 			const inputCode = fs.readFileSync(inputFile, 'utf-8');
 			const input = {
 				code: inputCode,
-				locate: getLocator(inputCode)
+				locate: getLocator(inputCode),
+				locate_1: getLocator(inputCode, { offsetLine: 1 })
```

Comment:
Should we call this something more meaningful like `source_map_locate`?

Reply:
no, its not related to sourcemap, locate returns line/column/character of a search string

Reply:
When do we use `locate` vs `locate_1`? Is there something more descriptive we can call tehse? I was assuming we need the 1-based line numbers only when dealing with source maps and otherwise use `locate`

Reply:
`locate_1` allows to write less code in combination with `mapConsumer` - its explained in line 100

```js
			// use locate_1 with mapConsumer:
			// lines are one-based, columns are zero-based
```